src/makeRobot.js

```diff
@@ -38,7 +38,59 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'cheap version': version,
+    'coords':
+    {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${this.name}, chip version: `
+      + `${this['cheap version']}, wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(n = 1) {
+      if (n >= 0) {
```

Comment:
```suggestion
      if (n > 0) {
```
0 никак не поменяет значение